index.html

```diff
@@ -15,6 +15,7 @@
     <div id="app">
       <div data-view="catalog"></div>
       <div data-view="details"></div>
+      <div data-view="cart"></div>
```

Comment:
We are going to use `data-view="cart"` for a future feature. Instead, use something like `#id="cart-count"`.